python_examples/R0_inference/learn_R.ipynb

```diff
@@ -0,0 +1,2500 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "2a92b1cb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "SI_mean = 4.7\n",
+    "SI_sd = 2.9"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fe6d4d08",
+   "metadata": {},
+   "source": [
+    "We can also use multiple serial intervals."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "f944a7ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read serial interval\n",
+    "serial_interval = pd.read_csv(\"si-covid.csv\", header=None)\n",
+    "serial_interval = serial_interval.fillna(0)\n",
+    "serial_intervals = serial_interval.values.T"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "2b00dfd3",
+   "metadata": {},
+   "source": [
+    "We estimate the Rt profile for the simulations of New Zealand. The expected R0 in these simulations is 3.28. The output files are stored locally and are not provided on the GitHub repository due to their size."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "922b364f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv('../../../../NewZealand/output_NZ_relaxed.csv')\n",
+    "\n",
+    "df = df.drop([\"InfectionStatus.Exposed\",\n",
+    "              \"InfectionStatus.InfectASympt\",\n",
+    "              \"InfectionStatus.InfectGP\",\n",
+    "              \"InfectionStatus.InfectHosp\",\n",
+    "              \"InfectionStatus.InfectICU\",\n",
+    "              \"InfectionStatus.InfectICURecov\",\n",
+    "              \"InfectionStatus.Dead\"],\n",
+    "              axis=1)\n",
+    "df = df.groupby([\"time\"]).agg({\"InfectionStatus.Susceptible\": 'sum',\n",
+    "                               \"InfectionStatus.InfectMild\": 'sum',\n",
+    "                               \"InfectionStatus.Recovered\": 'sum'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "d37630e8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = df['InfectionStatus.Susceptible']\n",
+    "# first generate a list of infected cases\n",
+    "locally_infected_cases = [0] + list(-np.diff(x.values)[:30])\n",
+    "# then create a list of the number of imported cases per day\n",
+    "imported = [1]*len(locally_infected_cases)\n",
+    "\n",
+    "# Build the serial interval w_s\n",
+    "num_timepoints = len(locally_infected_cases)\n",
+    "ws_mean = SI_mean\n",
+    "ws_var = SI_sd**2\n",
+    "theta = ws_var / ws_mean\n",
+    "k = ws_mean / theta\n",
+    "w_dist = scipy.stats.gamma(k, scale=theta)\n",
+    "disc_w = w_dist.pdf(np.arange(num_timepoints))\n",
+    "\n",
+    "\n",
+    "# Same inference, but using the LocImpBranchProPosterior\n",
+    "tau = 3\n",
```

Comment:
How sensitive are the results to this parameter value? 
I'm not suggesting that we change away from tau=3, just that if it is causing a significant difference we ought to be more careful about stating upfront (here, and more importantly in the supplement of the paper) that our results are conditional on a choice we decided to make for the sliding window, and maybe justifying this choice in some way. Especially since tau=3 is not necessarily a common choice that is frequently appearing in past applications of this method.